src/erc721/extensions/enumerable/module.cairo

```diff
@@ -0,0 +1,176 @@
+// Source: https://github.com/Seraph-Labs/Cairo-Contracts/blob/cairo-2/src/tokens/erc721/extensions/enumerable.cairo
+
+#[starknet::contract]
+mod ERC721Enumerable {
+    use option::OptionTrait;
+    use integer::BoundedInt;
+    use traits::{TryInto, Into};
+    use starknet::ContractAddress;
+    use cairo_erc_3525::src5::module::SRC5;
```

Comment:
Can the ERC721 token be in a separate repo that we include as a dependency?
That way we can reuse our 721 without importing 3525.

Reply:
Yes, this is part of the remaining work to do